metricbeat/helper/windows/pdh/pdh_query_windows.go

```diff
@@ -43,8 +43,8 @@ type Counter struct {
 
 // Query contains the pdh.
 type Query struct {
-	handle   PdhQueryHandle
-	counters map[string]*Counter
+	Handle   PdhQueryHandle
```

Comment:
The Handle looks like something quite internal, maybe there is no need of making it public.

Reply:
I had to make it public because I am checking those values in the perfmon/reader tests.